lib/Basics/MemoryTypes/MemoryTypes.h

```diff
@@ -0,0 +1,67 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
This file later needs to be merged with Lars' latest changes related to cluster info memory tracking